helm-charts/stable/prow-control-plane/templates/plugins-ConfigMap.yaml

```diff
@@ -18,6 +18,9 @@ metadata:
   name: plugins
 data:
   plugins.yaml: |
+    {{- if eq (len .Values.repositories) 0 }}
+    plugins: []
```

Comment:
Can we do the same thing we did above where we don't define the key in both again, but rather do either output `[]` or the else block?